docs/docs/spark-procedures.md

```diff
@@ -972,4 +972,101 @@ CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id =>
+This can be the starting point to fully or incrementally copy an Iceberg table to a new location.
+
+!!! info
+    This procedure only stages rewritten metadata files and prepares a list of files to copy. The actual file copy is not part of this procedure.
+
+
+| Argument Name      | Required? | default                                        | Type   | Description                                                             |
+|--------------------|-----------|------------------------------------------------|--------|-------------------------------------------------------------------------|
+| `table`            | ✔️        |                                                | string | Name of the table                                                       |
+| `source_prefix`    | ✔️        |                                                | string | The existing prefix to be replaced                                      |
+| `target_prefix`    | ✔️        |                                                | string | The replacement prefix for `source_prefix`                              |
+| `start_version`    |           | first metadata.json in table's metadata log    | string | The name or path to the chronologically first metadata.json to rewrite  |
+| `end_version`      |           | latest metadata.json                           | string | The name or path to the chronologically last metadata.json to rewrite   |
+| `staging_location` |           | new directory under table's metadata directory | string | The output location for newly modified metadata files                   |
+
+
+#### Modes of operation:
+
+- Full Rewrite:
+
+By default, the procedure operates in full rewrite mode, rewriting all reachable metadata files 
+(this includes metadata.json, manifest lists, manifests, and position delete files), and returning all reachable files in the result copy plan.
+
+- Incremental Rewrite:
+
+The `start_version` and/or `end_version` arguments may be provided to limit the scope of rewrite.  
+Only metadata files added between `start_version` and `end_version` will be rewritten, and only files added in this range are returned in the copy plan.
+
+#### Output
+
+| Output Name          | Type   | Description                                                       |
+|----------------------|--------|-------------------------------------------------------------------|
+| `latest_version`     | string | Name of the latest metadata file rewritten by this procedure      |
+| `file_list_location` | string | Path to a csv file containing a mapping of source to target paths |
+
+##### File List Copy Plan
+The file contains the copy plan for all files added to the table between `startVersion` and `endVersion`:
+
+For each file, it specifies
+
+- Source Path:
+The original file path in the table, or the staging location if the file has been rewritten.
```

Comment:
Should this be the same format as the previous list:  Incremental Rewrite and Full Rewrite? 

If so, there should be a space between "Source Path" and the next line, otherwise it shows as one line?  

And if we want two lines as the previous list, we should also remove the colon after "Source Path" and "Target Path".  I think colon should only be for if its on the same line.